scripts/build_python.bat

```diff
@@ -0,0 +1,6 @@
+@echo off
+REM Should be copied to the build directory
```

Comment:
why?

Reply:
Because the source directory is estimated assuming it is copied into build directory